focuspoints.lrdevplugin/Utils.lua

```diff
@@ -70,6 +70,23 @@ function split(str, delim)
 end
 
 --[[
+-- Breaks a delimited string into a table of substrings and removes whitespace
+-- split("A B C,D E", " ") -> { "A", "B", "C,D", "E" }
+-- str - string to be broken into pieces
+-- delim - delimiter
+--]]
+function splitTrim(str, delim)
```

Comment:
good change. I wonder if we should just change the ```split``` method to always trim. I'm not sure of a use-case where we wouldn't want it to.